CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 10.3.0 (YYYY-MM-DD)
+
+### Enhancements
+* None.
+
+### Fixes
+* [RealmApp] Integrating a remote Sync changeset into the local Realm could result in an `Index out of range error`.
```

Comment:
It looks like the Core release has a few other fixes we should also mention. Sorry for not catching this earlier. https://github.com/realm/realm-core/blob/master/CHANGELOG.md#1033-release-notes

I would propose changing it to this:

```
* [RealmApp] Integrating a remote Sync changeset into the local Realm could result in an `Index out of range error`.
* Change notifications not firing when removing and adding an object with the same primary key within a transaction (Issue [#7098](https://github.com/realm/realm-java/issues/7098)).
* Race condition which would lead to "uncaught exception in notifier thread: N5realm15InvalidTableRefE: transaction_ended" and a crash when the source Realm was closed or invalidated at a very specific time during the first run of a collection notifier (Core issue [#3761](https://github.com/realm/realm-core/issues/3761), since v7.0.0).
* Deleting and recreating objects with embedded objects could fail (Core issue [#4240](https://github.com/realm/realm-core/pull/4240), since v10.0.0)
```

But feel free to do that after CI has run, no reason to retrigger it for a doc update.